src/Graze/Monolog/Handler/RaygunHandler.php

```diff
@@ -46,14 +46,14 @@ protected function write(array $record)
 
         if (isset($context['exception']) && $context['exception'] instanceof \Exception) {
             $this->writeException(
-                $record['formatted'], 
+                $record,
                 $record['formatted']['tags'], 
                 $record['formatted']['custom_data'], 
                 $record['formatted']['timestamp']
             );
         } elseif (isset($context['file']) && $context['line']) {
             $this->writeError(
-                $record['formatted'], 
+                $record,
```

Comment:
Mainly for consistency with the exception case, but I'll go ahead and make it `'formatted'` again as it could potentially be useful to be normalized (like if the file name were unicode or something)